workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -7,18 +7,37 @@ jest.mock('uuid', () => ({ v4: () => 'someId' }));
 
 import { DynamoDBClient, GetItemCommand, UpdateItemCommand } from '@aws-sdk/client-dynamodb';
 import { marshall } from '@aws-sdk/util-dynamodb';
-import { resourceTypeToKey } from '@aws/workbench-core-base';
+import { removeDynamoDbKeys, resourceTypeToKey } from '@aws/workbench-core-base';
+import DynamoDBService from '@aws/workbench-core-base/lib/aws/helpers/dynamoDB/dynamoDBService';
 import { mockClient } from 'aws-sdk-client-mock';
 import { Account } from '../models/account';
-import CostCenter from '../models/costCenter';
+import { CostCenter } from '../models/costCenters/costCenter';
+import { ListCostCentersRequestParser } from '../models/costCenters/listCostCentersRequest';
 import CreateCostCenter from '../models/createCostCenterRequest';
 import CostCenterService from './costCenterService';
 
 describe('CostCenterService', () => {
   const ORIGINAL_ENV = process.env;
-  let accountMetadata: Account;
-  const costCenterService = new CostCenterService({ TABLE_NAME: 'tableName' });
   const accountId = 'acc-someId';
+  const accountMetadata: Account = {
```

Comment:
We should use the parser whenever we've got one to make sure all the data that we're setting is valid at compile time and run time.

Reply:
This is specifically the `accountMetadata` and not the full account object itself. I don't think we have an actual model for `accountMedata`. This is the same code as on line 35. I just moved the code up a level so it can be shared with all tests.

Reply:
🤦 forgot this wasn't on my feature branch